src/corefx/System.Globalization.Native/localeStringData.cpp

```diff
@@ -224,6 +229,16 @@ extern "C" int32_t GetLocaleInfoString(const UChar* localeName, LocaleStringData
 		case PositiveInfinitySymbol:
 			status = GetLocaleInfoDecimalFormatSymbol(locale, DecimalFormatSymbols::kInfinitySymbol, value, valueLength);
 			break;
+		case ParentName:
+			if (strlen(locale.getCountry()) > 0)
```

Comment:
The C API exposes `uloc_getParent`, are these isomorphic or are there cases where ICU locale names can have more than one underscore and so `uloc_getParent` would do something different?